tsl/src/init.c

```diff
@@ -177,6 +177,7 @@ CrossModuleFunctions tsl_cm_functions = {
 	.recompress_chunk_segmentwise = tsl_recompress_chunk_segmentwise,
 	.get_compressed_chunk_index_for_recompression =
 		tsl_get_compressed_chunk_index_for_recompression,
+	.preprocess_query_hook = tsl_preprocess_query_hook,
```

Comment:
```suggestion
	.preprocess_query = tsl_preprocess_query,
```
On non-tsl code we use `preprocess_query`, so any reason to don't use the same name here under the tsl code???

Reply:
Renamed